src/Console/Deployer.php

```diff
@@ -157,12 +157,22 @@ private function generateArchive(SymfonyStyle $io, ProgressBar $progress) : void
         // Set correct permissions on the file
         $this->fs->chmod('.bref/output/.bref/bin', 0755);
         // Install our custom php.ini and merge it with user configuration
-        $this->buildPhpConfig(
+        $phpConfig = $this->buildPhpConfig(
             __DIR__  . '/../../template/php.ini',
             '.bref/output/.bref/php.ini',
             $projectConfig['php']['configuration'] ?? [],
             $projectConfig['php']['extensions'] ?? []
         );
+        // Remove unused extensions
+        foreach (glob('.bref/output/.bref/bin/ext/*.so') as $extensionFile) {
+            if ($extensionFile === '.bref/output/.bref/bin/ext/opcache.so') {
+                continue;
+            }
+            $extensionName = substr($extensionFile, strrpos($extensionFile, '/') + 1, -3);
+            if (!array_key_exists($extensionName, $phpConfig)) {
+                $this->fs->remove($extensionFile);
+            }
+        }
```

Comment:
Like the other parts in this file could you move this logic to a private function? (e.g. `$this->removeUnusedExtensions($phpConfig)`)